ui/context_processors.py

```diff
@@ -15,7 +15,7 @@ def api_keys(request):
         "APIKEYS": {
             "GOOGLE": settings.GOOGLE_API_KEY,
             "GOOGLE_ANALYTICS": settings.GA_TRACKING_ID,
+            "GOOGLE_ADWORDS": settings.ADWORDS_CONVERSION_ID,
```

Comment:
If we do that, we'll have to change how the environment variables are populated, which is a larger change that involves devops. In addition, in the template we access this variable in the `APIKEYS` dict, and that name provides additional context for what `GOOGLE_ADWORDS` means. We don't have namespaces in environment variables.